core/services/keystore/keys/aptoskey/account_test.go

```diff
@@ -0,0 +1,141 @@
+package aptoskey
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// Tests extracted from
+// https://github.com/aptos-labs/aptos-go-sdk/blob/5ee5ac308e5881b508c0a5124f5e0b8df27a4d40/internal/types/account_test.go
```

Comment:
IIRC floating comments like this are dropped by the doc tools because they don't have anything to attach them to. How about making this a package comment, up at the very top?

Reply:
Actually, that doesn't make sense since this is a test file. Maybe this is fine as-is :thinking:

Reply:
Makes sense to have it as a package comment in my opinion, looks better